### PR TITLE
Started defining schema for cubes and other placeholds

### PR DESCRIPTION
**Pull request by thriuin**
Started defining schema for cubes as well as adding placeholders for logic and config files

### CODE REVIEW

File: ckanext/stcndm/logic/__init__.py [hunk 1]
```diff
@@ -0,0 +1,702 @@
...
+    def __init__(self, extra_msg=None):
+        self.extra_msg = extra_msg
+
+    def __str__(self):
+        err_msgs = (super(ActionError, self).__str__(),
+                    self.extra_msg)
+        return ' - '.join([str(err_msg) for err_msg in err_msgs if err_msg])
+
+
+class NotFound(ActionError):
```
Comment (wardi) [inline ckanext/stcndm/logic/__init__.py:43]: suggest importing these from ckanapi or ckan directly. Having multiple exception classes with the same name will lead to problems.

File: ckanext/stcndm/logic/__init__.py [hunk 2]
```diff
@@ -0,0 +1,702 @@
...
+import ckan.lib.navl.dictization_functions as df
+import ckan.plugins as p
+
+from ckan.common import _, c
+
+log = logging.getLogger(__name__)
+_validate = df.validate
+
+
+class AttributeDict(dict):
```
Comment (wardi) [inline ckanext/stcndm/logic/__init__.py:20]: I suggest removing this class, it has very surprising behaviour for python (might be expected for JS, but this isn't JS)

Comment (wardi) [inline ckanext/stcndm/logic/__init__.py:20]: Oh, I see this whole file was just copied from ckan.  I will remove this from ckan core.

**Comment by thriuin** [comment]
For sure, in the **init**.py I just coped this over from the old extension to get it to quickly run with a another function. Let's clean it up and do it properly. Thanks again!

**Pull request merged**